cmd/stanza/root.go

```diff
@@ -86,114 +55,19 @@ func runRoot(command *cobra.Command, _ []string, flags *RootFlags) {
 	}()
 
 	// Build agent service
-	service, ctx, err := service.NewBuilder().
+	service, _, err := service.NewBuilder().
```

Comment:
Do we still need to even return a context? I see it's being ignored here. I think I originally just returned it for pprof. I'd be in favor or removing it from the return if it's not used.

Reply:
Removed, since it doesn't seem like we use the context anywhere.